api/api.proto

```diff
@@ -830,6 +830,193 @@ service OpenStorageVolume {
     }
 }
 
+service OpenStorageCredentials {
```

Comment:
Spacing needs to be fixed in the file. Looks like some are tabs and some are spaces. Looks like the original file is created with tabs as spaces and tabs set to 2

Reply:
fixed